inst/include/h_util.hpp

```diff
@@ -28,6 +28,6 @@
  */
 #define MODEL_VERSION "2.5.0"
```

Comment:
Why don't you update this to 3.0 while you're at it?

Reply:
Sounds good, it has been updated & I added the v3.0 git tag

Reply:
That feels momentous 👏